gui/default/syncthing/core/logViewerModalView.html

```diff
@@ -29,6 +29,13 @@
     </div>
   </div>
   <div class="modal-footer">
+    <div>
+      <input type="text" id="logSearchInput" class="form-control" style="margin-bottom: 10px;" ng-keyup="logging.filter_input($event)">
```

Comment:
Please add a label to indicate what the input field is about. I think that in this case just "Filter" will be sufficient. There already is a similar filter in the Restore Versions modal, e.g.

https://github.com/syncthing/syncthing/blob/768fd6bff83447bc5a082b16af021507a7c0255f/gui/default/syncthing/folder/restoreVersionsModalView.html#L20-L23

Reply:
Latest commit has this label added, I also went ahead and wrapped them in a form-group following the convention in `restoreModalView`.